test/scripts/test-integration.sh

```diff
@@ -11,8 +11,10 @@ echo "Starting coverage tests..."
 # Ensure environment is test
 source "${SCRIPT_DIR}/env.sh"
 
-# Running in CI
 echo "Running in CI environment..."
+if [ ! -f .env ]; then
+  cp ./test/.env-func  .env
+fi
```

Comment:
we currently use `test/int/test-env-setup.js` to set env vars for integration tests.
with that in place, this change alone might not behave as expected, it could even lead to some unpredictable results

Reply:
@geka-evk, I believe it's this one [test/int/test-env-setup.js](https://github.com/pm4ml/connection-manager-api/blob/main/test/int/test-env-setup.js)